tests/testthat/test-compactify.R

```diff
@@ -124,7 +124,7 @@ quantile_pred_once <- function(estimates_vec, levels_vec) {
   hardhat::quantile_pred(t(as.matrix(estimates_vec)), levels_vec)
 }
 test_that("compactify works on distributions", {
-  skip("Until #611 is merged or hardhat/epipredict is patched")
+  skip("See #631.")
```

Comment:
weird. Dan wrote both, so you may just want to talk to him about it

Reply:
Chatted with him a bit.  Submitted a PR to hardhat to fix this + some other issues & then we can remove a partial fix sitting in epipredict.